schemas/stsci.edu/asdf/wcs/step-1.2.0.yaml

```diff
@@ -0,0 +1,31 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/wcs/step-1.2.0"
```

Comment:
Since 1.1.0 has not been released isn't it ok to accumulate changes and not bump to 1.2.0?